content/docs/docs-nav.yaml

```diff
@@ -62,35 +62,44 @@ nav:
   - In-depth:
       - API docs:
-        - Custom renderers: '/docs/details/custom-api-docs-renderers/'
-        - Using grpc-docs: '/docs/details/grpc-docs/'
+          - Custom renderers: '/docs/details/custom-api-docs-renderers/'
+          - Using grpc-docs: '/docs/details/grpc-docs/'
       - Configure GitHub Enterprise: '/docs/details/github-enterprise/'
       - GitHub app permissions: '/docs/details/github-app-permissions/'
       - Networking:
-        - Accessing AWS resources: '/docs/details/accessing-aws-resources/'
-        - Accessing public resources: '/docs/details/backend-reading-allow-list/'
-        - Allowlisting Roadie traffic: '/docs/details/allowlisting-roadie-traffic/'
-        - How Roadie connects: '/docs/details/how-roadie-connects/'
+          - Accessing AWS resources: '/docs/details/accessing-aws-resources/'
+          - Accessing public resources: '/docs/details/backend-reading-allow-list/'
+          - Allowlisting Roadie traffic: '/docs/details/allowlisting-roadie-traffic/'
+          - How Roadie connects: '/docs/details/how-roadie-connects/'
       - The catalog:
-        - Managing locations: '/docs/details/location-management/'
-        - Modelling dependencies: '/docs/details/showing-dependencies/'
-        - Removing components: '/docs/details/unregister-components/'
-        - Previewing component changes: '/docs/details/previewing-changes/'
-        - Troubleshooting the catalog: '/docs/details/troubleshooting-the-catalog/'
-        - Validating catalog files: '/docs/details/validator/'
-        - Ownership in Backstage Entities: '/docs/details/ownership/'
+          - Managing locations: '/docs/details/location-management/'
+          - Modelling dependencies: '/docs/details/showing-dependencies/'
+          - Removing components: '/docs/details/unregister-components/'
+          - Previewing component changes: '/docs/details/previewing-changes/'
+          - Troubleshooting the catalog: '/docs/details/troubleshooting-the-catalog/'
+          - Validating catalog files: '/docs/details/validator/'
+          - Ownership in Backstage Entities: '/docs/details/ownership/'
       - Testing page configurations: '/docs/details/sandbox-mode/'
       - Working with plugins:
-        - Required annotations: '/docs/details/required-annotations/'
-        - Requesting plugins: '/docs/details/requesting-plugins/'
+          - Required annotations: '/docs/details/required-annotations/'
+          - Requesting plugins: '/docs/details/requesting-plugins/'
       - Setting secrets: '/docs/details/setting-secrets/'
       - Updating the UI: '/docs/details/updating-the-ui/'
+  - Custom plugins:
+      - Overview: '/docs/custom-plugins/overview'
+      - Getting Started: '/docs/custom-plugins/getting-started/'
+      - Developing: '/docs/custom-plugins/developing'
+      - Registering Plugins: '/docs/custom-plugins/plugin-registration'
+      - Deploying: '/docs/custom-plugins/deploying'
+      - Securing Connections:
```

Comment:
I think this should be called something like "Connecting back to your network". That's the objective achieved by reading these pages. Just like "Deployment" is the objective achieved by the deployment page.